OasisDEX/oasis-borrow
# Info cards on homepage

# [Info cards on homepage](https://app.shortcut.com/oazo-apps/story/3228/create-homepage)
  
## Changes 👷‍♀️
- adds info cards to homepage (also mobile view)
- removes unnecessary ProductCardLayout component
- fixes text wrapping issue when Product description is long on product cards:
  
## How to test 🧪
- open homepage
- observe the info cards at the bottom
- resize the viewport and make sure they collapse properly
    
## Definition of done ✔️
